src/config/config.cc

```diff
@@ -189,6 +189,7 @@ Config::Config() {
       {"json-max-nesting-depth", false, new IntField(&json_max_nesting_depth, 1024, 0, INT_MAX)},
       {"json-storage-format", false,
        new EnumField<JsonStorageFormat>(&json_storage_format, json_storage_formats, JsonStorageFormat::JSON)},
+      {"lite-mode", false, new YesNoField(&lite_mode, false)},
```

Comment:
It should be necessary to add an option to enable this since it's safe to shrink the infrequently used column families. cc @torwig @PragmaTwice

Reply:
@git-hulk Hi，I submit a commit in this pr to support dynamic configuration of this option，I'm not sure that's what you mean，please review